package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "web3-onboard-monorepo",
-  "version": "2.1.0",
```

Comment:
@Adamj1232 wondering if we should always use minor instead of the patch version for the base version? I think patch is reserved for bug fixes? and minor adds new functionality

Reply:
@Adamj1232 disregard moving comment to main